README.md

```diff
@@ -2,12 +2,19 @@
 
 This tool manages [Lmod](https://github.com/TACC/Lmod) modulefiles for various AdaCore software. The created modulefiles enable easy switching between different software versions. Lmod achieves that by dynamically changing the `PATH` environment variable.
 
+## Installation
+
+```console
+$ git clone https://github.com/AdaCore/lmod_manager.git
+$ pip install lmod_manager
```

Comment:
Would there be any problem to propose `pip3 install git+https://github.com/AdaCore/lmod_manager.git` instead?

Reply:
I don't think so. The approach you suggest is much more elegant. I have changed the description.